### ISSUE TITLE
NTR: S-adenosylhomocysteine hydrolase activity low in RBC (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8304]
New term request S-adenosylhomocysteine hydrolase activity low in RBC (elthat_130118072107): AdoHcyase; SAHase; erythrocytes, red blood cells

**Comment by pnrobinson** [comment]
@MickeySegal
if this is 613752 , the activity is not specific to RBCs, and I would make a "tissue" term. Is this appropriate?

**Comment by MickeySegal** [comment]
Our original source of https://www.ncbi.nlm.nih.gov/books/NBK1483/ no longer mentions this, but other sources mention it, but https://omim.org/entry/613752 mentions liver testing so a general tissue finding seems right.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000750
- label: Decreased tissue S-adenosylhomocysteine hydrolase activity
- parents: [Abnormal tissue enzyme concentration or activity (HP:6000231)](https://hpo.jax.org/app/browse/term/HP:6000231)
- pmid: PMID:15024124
- def: Concentration or activity of S-adenosylhomocysteine hydrolase (EC 3.3.1.1) below the lower limit of normal. FECH enzyme can be measured in multiple tissues including erythrocytes and liver.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8304
- synonyms:
